a10_octavia/common/config_options.py

```diff
@@ -42,6 +42,9 @@
     cfg.IntOpt('default_axapi_version',
                default=30,
                help=_('VThunder axapi version')),
+    cfg.StrOpt('partition',
```

Comment:
This instead needs to be relegated solely to the rack device options

Reply:
removing this